src/content/reference/okteto-cli.mdx

```diff
@@ -7,7 +7,12 @@ id: okteto-cli
 
 import TiersList from "@theme/TiersList";
 
-The Okteto Command Line Interface is a unified tool to manage Development Environments.
+The Okteto Command Line Interface (CLI) is a unified tool to manage your applications in Okteto directly from your terminal.
+Follow this [guide to download and install the CLI](get-started/install-okteto-cli.mdx) if you don't already have it. The CLI is an [open-source](https://github.com/okteto/okteto) tool that can be used independently or integrated with the Okteto Platform.
+- Commands labeled `Platform` are available exclusively on the Okteto platform and require the Okteto Helm Chart to be installed in your Kubernetes cluster
```

Comment:
Should we add a similar explanations (and the labels) to the Okteto Manifest reference. Only the `dev` section is supported